rocketpy/rocket/parachute.py

```diff
@@ -170,6 +197,15 @@ def __init__(
         self.clean_pressure_signal_function = Function(0)
         self.noisy_pressure_signal_function = Function(0)
         self.noise_signal_function = Function(0)
+        self.radius = radius
+        self.height = height or radius
+        self.porosity = porosity
+        self.ka = 1.068 * (
+            1
+            - 1.465 * self.porosity
+            - 0.25975 * self.porosity**2
+            + 1.2626 * self.porosity**3
+        )
```

Comment:
Could you document Parachute.ka in the class docstring please?

Also, what does ka mean? Could we think about a new, more descriptive name for it?

In python, we rather use the explicit version than implicit version of name variables.


self.aoa is not preferred
self.angle_of_attach is preferred